app/server/lib/Triggers.ts

```diff
@@ -786,6 +793,7 @@ export class DocTriggers {
           body,
           headers: {
             'Content-Type': 'application/json',
+            'authorization': authorization,
```

Comment:
1. I suggest to name the header `Authorization` with a capital `A` (cf how it is usually named in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization));
2. I also suggest to remove the header if the authorization is empty or undefined.

```suggestion
            ...(authorization ? {'Authorization': authorization} : {}),
```